packages/rn-tester/js/examples/TextInput/TextInputExample.ios.js

```diff
@@ -648,66 +649,71 @@ const textInputExamples: Array<RNTesterModuleExample> = [
     title: 'TextInput Intrinsic Size',
     render: function (): React.Node {
       return (
-        <View>
-          <Text>Singleline TextInput</Text>
-          <View style={{height: 80}}>
-            <TextInput
-              style={{
-                position: 'absolute',
-                fontSize: 16,
-                backgroundColor: '#eeeeee',
-                borderColor: '#666666',
-                borderWidth: 5,
-                borderTopWidth: 20,
-                borderRadius: 10,
-                borderBottomRightRadius: 20,
-                padding: 10,
-                paddingTop: 20,
-              }}
-              testID="singleline_textinput"
-              placeholder="Placeholder defines intrinsic size"
-            />
-          </View>
-          <Text>Multiline TextInput</Text>
-          <View style={{height: 130}}>
-            <TextInput
-              style={{
-                position: 'absolute',
-                fontSize: 16,
-                backgroundColor: '#eeeeee',
-                borderColor: '#666666',
-                borderWidth: 5,
-                borderTopWidth: 20,
-                borderRadius: 10,
-                borderBottomRightRadius: 20,
-                padding: 10,
-                paddingTop: 20,
-                maxHeight: 100,
-              }}
-              testID="multiline_textinput"
-              multiline={true}
-              placeholder="Placeholder defines intrinsic size"
-            />
-          </View>
+        <RNTesterThemeContext.Consumer>
```

Comment:
Thanks for the fix!

I see there is also `LabelColor` that should maybe be used for the text itself?

Also, consider getting via `useContext` instead of the context component, for readability.

Reply:
Thank you for your feedback, I'll be changing it tomorrow.